subprojects/core/src/main/java/org/gradle/initialization/ParallelismBuildOptionFactory.java

```diff
@@ -56,34 +56,23 @@ public void applyTo(boolean value, ParallelismConfiguration settings, Origin ori
     public static class MaxWorkersOption extends StringBuildOption<ParallelismConfiguration> {
```

Comment:
There are definitely other implementations of `BuildOption` that need to adapt to the same pattern. Their tests should be changed accordingly.

Reply:
Sure, I believe I finished all of them.

Reply:
:x: We still need to use it in `LoggingConfigurationBuildOptionFactory.LogLevelOption` and `LoggingConfigurationBuildOptionFactory.ConsoleOption`.

Reply:
Oops.